Synview-backend/utils/JWTHelpers.ts

```diff
@@ -0,0 +1,41 @@
+import { Context } from "../deps.ts";
+export default async function Token(
+  context: Context,
+  next: () => Promise<unknown>
+) {
+  try {
+    const token = localStorage.getItem("token");
+    if (!token) {
+      return;
+    }
+    context.response.headers.set("Authorization", `Bearer ${token}`)
+    await next();
+  } catch (e) {
+    throw new Error("Didnt find any auth token: " + e);
+  }
+}
```

Comment:
I am a bit confused with this - this seems unused? There is a bunch of JWT code living in `user_routes.ts`, but the purpose of a Helper file like this should be to isolate/contain all of the JWT specific code to this file